### PR TITLE
Add failing test fixture for RemoveAlwaysElseRector

### PR DESCRIPTION
**Pull request by staabm**
# Failing Test for RemoveAlwaysElseRector

Based on https://getrector.org/demo/785003f6-cc99-4fdb-9f55-a26be24e0bfd

covers https://github.com/rectorphp/rector/issues/6505

### CODE REVIEW

**Comment by samsonasik** [comment]
could you provide a patch as well, there is mirrorComments method for it https://github.com/rectorphp/rector-src/blob/c8900bf34845ea694b84443c6ccfc09e394f3e89/src/Rector/AbstractRector.php#L371

File: rules-tests/EarlyReturn/Rector/If_/RemoveAlwaysElseRector/Fixture/lost_comment_before_elseif2.php.inc [hunk 1]
```diff
@@ -22,4 +22,31 @@ class StringConverter
...
+        if (is_array($data)) {
+
+            return $res;
+        
+        }
+        
+        if ($data instanceof EncodedString) {
+        }
+        
+        // this comment will be lost :-(
```
Comment (samsonasik) [inline rules-tests/EarlyReturn/Rector/If_/RemoveAlwaysElseRector/Fixture/lost_comment_before_elseif2.php.inc]: should be "this comment should be kept"

**Comment by staabm** [comment]
I guess the problem lies in `addNodeBeforeNode` and/or `addNodesAfterNode` which seems to be outside of the scope of this actual rector?

**Comment by staabm** [comment]
I have no idea how a possible fix could/can look like and would welcome any pointers

**Comment by samsonasik** [comment]
@staabm I cherry-picked your commit at PR https://github.com/rectorphp/rector-src/pull/234

**Pull request closed without merging**